game/players.cc

```diff
@@ -131,7 +131,7 @@ void Players::filter_internal() {
 			});
 		}
 // 		for (auto const& p: m_players) {
-// 			if (regex_search(p.name, regex(m_filter, regex_constants::icase))) filtered.push_back(p);
+// 			if (regex_search(p.name, regex(m_filter, std::regex_constants::icase))) filtered.push_back(p);
 // 		}
```

Comment:
Hmmm why did we comment this code out? Let's remove it if we don't intend to use player filtering.

Reply:
done